examples/stubbing-spying__route2/cypress/integration/stub-fetch-spec.js

```diff
@@ -0,0 +1,133 @@
+/// <reference types="Cypress" />
+
+describe('route2', () => {
+  context('stubbing', function () {
+    it('shows no Response message', () => {
+      // stub server response with []
+      cy.route2('/favorite-fruits', JSON.stringify([]))
```

Comment:
does this not work? it will automatically set content-type application/json for you

```suggestion
      cy.route2('/favorite-fruits', [])
```

Reply:
opened a bug report: https://github.com/cypress-io/cypress/issues/8532